packages/astro/src/assets/fonts/providers/adobe.ts

```diff
@@ -0,0 +1,9 @@
+import { defineFontProvider } from '../helpers.js';
+
+export function adobe(config: { apiKey: string }) {
+	return defineFontProvider({
+		name: 'adobe',
+		entrypoint: 'astro/assets/fonts/providers/adobe',
```

Comment:
Entrypoints should have the file extension. E.g. `astro/assets/fonts/providers/adobe.js`